### ISSUE TITLE
Postgres 10.x not supported in metric-postgres-graphite.rb

### ISSUE THREAD

**Issue by jfineberg** [sensu-plugins/sensu-plugins-postgres#52]
Postgres has updated the names of the SQL functions for checking replication status in 10.x

https://www.postgresql.org/docs/10/static/release-10.html

```
Rename SQL functions, tools, and options that reference “xlog” to “wal” (Robert Haas)

For example, pg_switch_xlog() becomes pg_switch_wal(), pg_receivexlog becomes pg_receivewal, and --xlogdir becomes --waldir. This is for consistency with the change of the pg_xlog directory name; in general, the “xlog” terminology is no longer used in any user-facing places.
```

`check-postgres-replication.rb` seems to be aware of this, and performs a version check, however `metric-postgres-graphite.rb` does not and will simply error in Postgres 10.x.

```
Check failed to run: ERROR:  function pg_current_xlog_location() does not exist
LINE 1: SELECT pg_current_xlog_location()
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
, ["/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-postgres-1.4.5/bin/metric-postgres-graphite.rb:95:in `exec'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-postgres-1.4.5/bin/metric-postgres-graphite.rb:95:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugin-1.4.5/lib/sensu-plugin/cli.rb:58:in `block in <class:CLI>'"]
```

**Comment by phumpal** [comment]
@jfineberg @majormoses IIRC this can be closed.

**Comment by majormoses** [comment]
fixed via #69
